tools/cpplint.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/env python
```

Comment:
Oh, right. I have a `python2` and a `python2.7`. `python2` would also have been my choice, but whatever is consistent is fine.